src/lib/eventStack/EventStack.js

```diff
@@ -0,0 +1,70 @@
+import isBrowser from '../isBrowser'
+import EventTarget from './EventTarget'
+import normalizeHandlers from './normalizeHandlers'
+import normalizeTarget from './normalizeTarget'
+
+export default class EventStack {
```

Comment:
It would be helpful to have a description above the EventSet, EventStack, and EventPool.  It is not immediately clear how the 3 operate together.

Reply:
@levithomason I've added a README file with description